rest-api-spec/src/main/resources/rest-api-spec/api/indices.create.json

```diff
@@ -13,6 +13,10 @@
         }
       },
       "params": {
+        "wait_for_active_shards": {
```

Comment:
what about the other rest actions that support this parameter now? e.g. shrink.

Can you also add a rest test?